.github/CODEOWNERS

```
@@ -0,0 +1,11 @@
+# CODEOWNERS file.
+# See: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+./common @alber70g
+./*.*      @alber70g
+
+./common @randynamic
+./*.*      @randynamic
+
+./common @ggobugi27
+./*.*      @ggobugi27
```

These can be combined to one line

```suggestion
./common @alber70g @randynamic @ggobugi27
./*.*      @alber70g @randynamic @ggobugi27 

```

And it's probably better to have a team name there. The owner of the repo can setup teams inside GitHub. Then you'd be able to say `./common @kadena-community/team` I think

---

Yes when I get access I'll create the teams
short-term sollution